public/style/components/_card.scss

```diff
@@ -0,0 +1,14 @@
+.card {
```

Comment:
Possibly merge this into `_structure.scss`? I don't see there being a lot more styles in each one.